homeassistant/components/recorder/statistics.py

```diff
@@ -776,7 +774,7 @@ def _update_statistics(
 
 
 def _generate_get_metadata_stmt(
-    statistic_ids: list[str] | tuple[str] | None = None,
+    statistic_ids: list[str] | tuple[str, ...] | None = None,
```

Comment:
Based on the documentation, I think you should drop tuple altogether:
```suggestion
    statistic_ids: list[str] | None = None,
```